### PR TITLE
Add and update UK Police Forces

### PR DESCRIPTION
**Pull request by tomodwyer**
**Renamed rules for the following UK Police Forces:**

- [Cambridgeshire Constabulary](https://www.cambs.police.uk/)
- [Essex Police](https://www.essex.police.uk/)
- [Hertfordshire Constabulary](https://www.herts.police.uk/)
- [Staffordshire Police](https://www.staffordshire.police.uk/)
- [West Midlands Police](https://www.west-midlands.police.uk/)

to match official names: [https://www.police.uk/forces/](https://www.police.uk/forces/)

**Added rules for the following UK Police Forces:**

- [Avon and Somerset Constabulary](https://www.avonandsomerset.police.uk/)
- [City of London Police](https://www.cityoflondon.police.uk/)
- [Cleveland Police](https://www.cleveland.police.uk/)
- [Cumbria Constabulary](https://www.cumbria.police.uk/)
- [Devon and Cornwall Police](https://www.devon-cornwall.police.uk/)
- [Durham Constabulary](https://www.durham.police.uk/)
- [Gloucestershire Constabulary](https://www.gloucestershire.police.uk/)
- [Leicestershire Police](https://www.leics.police.uk/)
- [Warwickshire Police](https://www.warwickshire.police.uk/)
- [West Mercia Police](https://www.westmercia.police.uk/)

**Added a rule for the main [Police UK website](https://www.police.uk/).**

### CODE REVIEW

**Comment by J0WI** [comment]
I'm not sure if they should be merged into a single *.police.uk rule. Are there many others or some fourth and fifth level subdomains?

**Comment by tomodwyer** [comment]
There are two problems with a single rule: 

1. Many of the forces don't use https on their site and there is no current legal mandate forcing them to use https
1. Each force controls their own website, so without manually checking each site, I cannot be sure of any fourth or fifth level subdomains

File: src/chrome/content/rules/AvonandSomersetConstabulary.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Avon and Somerset Constabulary">
+	<target host="avonandsomerset.police.uk" />
+	<target host="www.avonandsomerset.police.uk" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/AvonandSomersetConstabulary.xml]: please also add
avonandsomerset.police.uk
www.avonandsomerset.police.uk

File: src/chrome/content/rules/CityofLondonPolice.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="City of London Police">
+	<target host="cityoflondon.police.uk" />
+	<target host="www.cityoflondon.police.uk" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/CityofLondonPolice.xml]: Please also add:
academy.cityoflondon.police.uk

Mismatch:
wwwstage.cityoflondon.police.uk
editcolp.cityoflondon.police.uk

File: src/chrome/content/rules/DevonandCornwallPolice.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Devon and Cornwall Police">
+	<target host="devon-cornwall.police.uk" />
+	<target host="www.devon-cornwall.police.uk" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/DevonandCornwallPolice.xml]: WFM:
services.devon-cornwall.police.uk

Refused:
dogsection.devon-cornwall.police.uk
digitalmediablog.devon-cornwall.police.uk

File: src/chrome/content/rules/LeicestershirePolice.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Leicestershire Police">
+	<target host="leics.police.uk" />
+	<target host="www.leics.police.uk" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/LeicestershirePolice.xml]: Mismatch:
helicopter.leics.police.uk

File: src/chrome/content/rules/HertfordshireConstabulary.xml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<ruleset name="Herts.police.uk">
+<ruleset name="Hertfordshire Constabulary">
   <target host="herts.police.uk" />
   <target host="www.herts.police.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/HertfordshireConstabulary.xml]: Mismatch:
snt.herts.police.uk

File: src/chrome/content/rules/WestMidlandsPolice.xml [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<ruleset name="West-midlands.police.uk">
```
Comment (J0WI) [inline src/chrome/content/rules/WestMidlandsPolice.xml]: Please add a hint for the 404 on ^

File: src/chrome/content/rules/WestMidlandsPolice.xml [hunk 7]
```diff
@@ -1,4 +1,4 @@
...
+<ruleset name="West Midlands Police">
   <target host="west-midlands.police.uk" />
   <target host="www.west-midlands.police.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/WestMidlandsPolice.xml]: WFM:
foi.west-midlands.police.uk

mixed content (please check for broken content):
behindthecrime.west-midlands.police.uk
jobs.west-midlands.police.uk
saferstudents.west-midlands.police.uk
safestreet.west-midlands.police.uk

File: src/chrome/content/rules/EssexPolice.xml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<ruleset name="Essex.police.uk">
```
Comment (J0WI) [inline src/chrome/content/rules/EssexPolice.xml]: Please add a hint for the mismatch on ^.

File: src/chrome/content/rules/CumbriaConstabulary.xml [hunk 9]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Cumbria Constabulary">
+	<target host="cumbria.police.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/CumbriaConstabulary.xml]: Got 404 on ^. Please redirect to www and add a hint.

File: src/chrome/content/rules/ClevelandPolice.xml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Cleveland Police">
```
Comment (J0WI) [inline src/chrome/content/rules/ClevelandPolice.xml]: This hosts ask me for a client cert. That's weird.
rule lgtm.

File: src/chrome/content/rules/PoliceUK.xml [hunk 11]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Police UK">
+	<target host="police.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/PoliceUK.xml]: Timeout on ^, please redirect and comment.

**Comment by J0WI** [comment]
Please link all the rulesets in a comments to the police.uk one.

**Comment by tomodwyer** [comment]
Alright I'm on it :+1:

**Comment by J0WI** [comment]
Great, all rules lgtm now. Just a minor change in comments left.

**Comment by tomodwyer** [comment]
OK those comments have been updated. Let me know if there are any other changes you would like :smiley:

**Comment by J0WI** [comment]
file names means to also include the `.xml` extension. ;)

**Comment by tomodwyer** [comment]
Thanks. That's fixed now :+1:

**Comment by J0WI** [comment]
Very nice, thanks for all the new rules!

**Pull request merged**